tests/integrations/controllers/api/UserResource.py

```diff
@@ -0,0 +1,17 @@
+from ...app.User import User
+from src.masonite.api.resources import Resource
+
+
+class UserResource(Resource):
+    def show(self, request, user_id):
+        return User.find(user_id)
+
+    def index(self, request):
+        return User.paginate(request.input("limit", 10), request.input("page", 1))
+
+    def to_dict(self, request, model):
```

Comment:
When seeing this I guess it covers lots of use cases but it would be also cool to be able to have a "quickest alternative" where we would specify fields only as in Django Rest Framework.
Something like this:
```
# ...
     return ["name", "email"]

or __fields__ = ["name", "email"]
```
a bit like visible and hidden attrs on ORM models definition